.gitignore

```diff
@@ -4,3 +4,4 @@
 .task
 
 src/Makevars
+flint
```

Comment:
Can this be removed, at least at this time?
You can use `.git/info/exclude` or something.

Reply:
Nice trick, I didn't know that